src/components/AnnotationCreator.jsx

```diff
@@ -188,26 +251,43 @@ export default function AnnotationCreator({
           </Select>
         </FormControl>
       </div>
-      {error && (
-        <CustomAlert
-          titleId="SERVER_ERROR"
+      {hasErrors && (
+        <Alert
+          severity="error"
+          titleId="AN_ERROR_OCCURRED"
           style={{
             margin: '16px auto 8px auto',
             maxHeight: '80vh',
             width: 600,
             padding: '0 40px',
           }}
-          severity="error"
         >
-          {error}
-        </CustomAlert>
+          {addEncounterToSightingError && (
+            <Text key={addEncounterToSightingError} variant="body2">
+              {addEncounterToSightingError}
+            </Text>
+          )}
+          {addAnnotationsToSightingEncounterError && (
+            <Text
+              key={addAnnotationsToSightingEncounterError}
```

Comment:
keys shouldn't be necessary unless you are iterating. were you getting a key error?